### PR TITLE
PHC-4753 Add comment count and ReactionToolbar

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - Adds comment count
  - Adds reaction toolbar for viewing reactions and reacting to posts

## Todo

We optimistically assume that the mutation will succeed and capture the reaction locally. Need to add handling for when the mutation fails to:
A) alert the user to the failure
B) refetch the posts from the server to ensure we are showing the actual reactions

### Example app
https://user-images.githubusercontent.com/76954025/236299699-61eba035-4670-4fb5-a8fe-981c740cd0ea.mp4

### Starter app
https://user-images.githubusercontent.com/76954025/236299379-a2c4fa1f-2236-4f7d-b5c3-c89624f87f3d.mp4

### CODE REVIEW

File: src/components/Circles/ReactionsToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    height: '80%',
+  },
+  reactionContainer: {
+    paddingRight: theme.spacing.micro,
+  },
+  reactionContent: {
+    height: 30,
+  },
+  reactionLabel: {
+    paddingTop: 4,
```
Comment (bruce-glazier) [inline src/components/Circles/ReactionsToolbar.tsx:135]: Using fixed padding values as this correctly positions the text within the button and we wouldn't want these to change.

File: src/components/Circles/ReactionsToolbar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+useUserMock.mockReturnValue({
+  data: {
+    id: 'userId',
+  },
+});
+
+const baseURL = 'https://some-domain/unit-test';
+const toolbarComponent = (post: ActivePost) => (
+  <QueryClientProvider client={new QueryClient()}>
+    <GraphQLClientContextProvider baseURL={baseURL} />
```
Comment (sternetj) [inline src/components/Circles/ReactionsToolbar.test.tsx:34]: Should this wrap `ReactionsToolbar`?

**Review by sternetj — APPROVED** [review]
Looks Awesome!

File: src/components/Circles/ReactionsToolbar.test.tsx (hunk 2 above)
Comment (bruce-glazier) [reply src/components/Circles/ReactionsToolbar.test.tsx:34]: I would have thought so but it seems to be working as expected 🤔 I'll take a look at that and address if needed in another PR.

**Pull request merged**